lib/bgpstream_elem.h

```diff
@@ -215,6 +215,12 @@ typedef struct bgpstream_elem {
    */
   bgpstream_as_path_t *as_path;
 
+  /** Prefix originator AS number
+   *
+   * Available only for RIB and Announcement elem types
+   */
+  int32_t origin_asn;
```

Comment:
what is this for?

Reply:
ah, not useful anymore. will remove it in next commit.

Reply:
removed in https://github.com/CAIDA/libbgpstream/pull/46/commits/fb2f8e6eb59d726138f3609de3051aba1092b8c6